tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -347,6 +350,74 @@ void SetHtmlResponseContent()
             return blockingAction;
         }
 
+        private string GetJsonResponse()
+        {
+            try
+            {
+                if (_blockedJsonTemplateCache != null)
+                {
+                    return _blockedJsonTemplateCache;
+                }
+
+                if (_settings.BlockedJsonTemplatePath != null && AppDomain.CurrentDomain.BaseDirectory != null)
+                {
+                    var fullPath =
+                        Path.IsPathRooted(_settings.BlockedJsonTemplatePath)
+                            ? _settings.BlockedJsonTemplatePath
+                            : Path.Combine(AppDomain.CurrentDomain.BaseDirectory, _settings.BlockedJsonTemplatePath);
```

Comment:
Makes me wonder, should we be concerned about directory traversal attacks in our settings? 🤔

Reply:
fixed in 68156022677fa23f7162678e322e888f850ddb46

Reply:
I believe the inputs should considered as trusted, as long as the settings can't be controlled by attackers, a directory traversal shouldn't be an issue.

If the attackers control env var or config files, there's much worse that they can do.

If you disagree let's discuss off-line.